bitnami/wordpress/values.yaml

```diff
@@ -14,9 +14,6 @@ global:
   ##   - myRegistryKeySecretName
   ##
   imagePullSecrets: []
-  ## E.g.
-  ## storageClass: myStorageClass
-  ##
```

Comment:
Why is this required?

Reply:
Our internal processes only match the first occurrence. Commented or not, so if we add both commented and no commented it will duplicate the line

Reply:
Then, this is a limitation to be solved in the internal process, not here.

By doing this, the final product (the chart in this case) is the one "harmed". I know it's harder to fix this in the internal automations but that's not an excuse since the product shouldn't be adapted to fit in the pipeline but the other way around.